mono/utils/mono-time.c

```diff
@@ -109,6 +108,7 @@ mono_100ns_datetime (void)
 
 #include <time.h>
 
+#if 0
```

Comment:
Just delete it.   It should've been removed as part of 06a4273d73cfa732d911bdaf50f752c025a0665c

Reply:
Ok.